app/frontend/src/authConfig.ts

```diff
@@ -133,14 +133,15 @@ export const appServicesLogout = () => {
 
 // Determine if the user is logged in
 // The user may have logged in either using the app services login or the on-page login
-export const isLoggedIn = (client: IPublicClientApplication | undefined): boolean => {
-    return client?.getActiveAccount() != null || appServicesToken != null;
+export const checkLoggedIn = async (client: IPublicClientApplication | undefined): Promise<boolean> => {
+    return client?.getActiveAccount() != null || (await getAppServicesToken()) != null;
```

Comment:
I wonder if it would help developers debugging if this was a full if/else with a console.log for "Logged in with Built-in Auth on App Service", that sort of thing. I've started adding those sorts of logs in similar scenarios, i.e. whenever an env var affects an if/else, to help developers debug their setups.

Reply:
Did not add the console.log but took your advice on ifs